CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Fixes
 
+* `[jest-resolve]` Update node module resolution algorithm to follow physical paths ([#5085](https://github.com/facebook/jest/pull/5085))
```

Comment:
Suggest to rephrase as "Update node module resolution algorithm to correctly handle symlinked paths"